extensions/wikia/Forum/Forum.i18n.php

```diff
@@ -67,6 +67,8 @@
 	'forum-board-new-message-heading' => 'Start a Discussion',
```

Comment:
Changes in this file related to [SOC-590](https://wikia-inc.atlassian.net/browse/SOC-590).